test/trackr-lib.test.ts

```diff
@@ -0,0 +1,10 @@
+import * as trackr from '../src/app.ts'
```

Comment:
What is this file about?

Please, update commits to avoid to add it to the git history :-)